ui/browsing/artistpage.go

```diff
@@ -318,6 +321,25 @@ func NewArtistPageHeader(page *ArtistPage) *ArtistPageHeader {
 		go a.artistPage.contr.PlayArtistDiscography(a.artistID, false /*shuffle*/)
 	})
 	a.playRadioBtn = widget.NewButtonWithIcon("Play Artist Radio", myTheme.ShuffleIcon, a.artistPage.playArtistRadio)
+
+	_, canShare := a.artistPage.mp.(mediaprovider.SupportsSharing)
```

Comment:
Let's not enable sharing for artists for now. Even though Navidrome supports it, officially only albums and songs can be shared, so other Subsonic servers may not support it:

https://opensubsonic.netlify.app/docs/endpoints/createshare/

The proper way to support this would be to propose an extension in the OpenSubsonic org that allows clients to query whether the server supports an extended version of this API that can share artists and playlists as well. If not, what would also be acceptable is to check whether the server type is Navidrome and if so, enable the extended sharing features. But I think just waiting to get an OpenSubsonic extension approved for this might be the way to go.

Reply:
If you want you can just comment this code out and add a TODO, so we can easily add it back in the future.

Reply:
Good call. I added a `CanShareArtists()` method to the interface, in case we want to iteratively enable the feature depending on the Subsonic server (e.g. Navidrome).